client/template.go

```diff
@@ -188,3 +201,39 @@ func (client *ApiClient) AssignTemplateToProject(id string, payload TemplateAssi
 func (client *ApiClient) RemoveTemplateFromProject(templateId string, projectId string) error {
 	return client.http.Delete("/blueprints/" + templateId + "/projects/" + projectId)
 }
+
+func (client *ApiClient) VariablesFromRepository(payload *VariablesFromRepositoryPayload) ([]ConfigurationVariable, error) {
```

Comment:
This is a little annoying. But basically, the code converts a golang struct "payload" to URL params (a map of strings in golang).

(Since it's a "GET" and not a "POST").

1. From struct to JSON.
2. From JSON to map of strings.

Reply:
good idea! I'll see what I can do.

Reply:
moved some of the code to a utils.go file.